src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -254,25 +254,27 @@ export async function POST(request: NextRequest) {
           updateFromEvent,
         });
 
-        const refreshToken = subscriber.fxa_refresh_token;
-        const accessToken = subscriber.fxa_access_token;
-        if (refreshToken === null || accessToken === null) {
+        const refreshToken = subscriber.fxa_refresh_token ?? "";
+        const accessToken = subscriber.fxa_access_token ?? "";
+        if (accessToken === null || refreshToken === null) {
```

Comment:
I'm a bit confused. L257 is setting `refreshToken` to `subscriber.fxa_refresh_token` and then uses nullish coalescing to fall back to an empty string.
How would `refreshToken` or `accessToken` be null?

Reply:
```js
const subscriber = {};
let refreshToken = subscriber.fxa_refresh_token ?? "huh";
console.log(refreshToken); // 'huh'
```

Reply:
thanks @pdehaan, I was in a rush to get this in, I can take these out